configuring.tex

```diff
@@ -118,7 +120,7 @@ \section{Formatting SD cards}
 \item {\tt C64THUMB.M65} (C64 thumbnail image used in freezer)
 \item {\tt C65THUMB.M65} (C65 thumbnail image used in freezer)
 \item {\tt MEGA65.ROM}   (128KB ROM file)
-\item {\tt MEGA65.D81 (default disk image, automatically mounted)}
+\item {\tt MEGA65.D81} (Disk image)
```

Comment:
I took it out because the user has the option to change the default disk image, and didnt want them thinking that this image was _always_ the default.